client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -0,0 +1,56 @@
+import { translate } from 'i18n-calypso';
+import { Task } from './types';
+
+export function getEnhancedTasks( tasks: Task[], siteSlug: string | null ) {
+	const enhancedTaskList: Task[] = [];
+	tasks &&
+		tasks.map( ( task ) => {
+			let taskData = {};
+			switch ( task.id ) {
+				case 'setup_newsletter':
+					taskData = {
+						title: translate( 'Set up Newsletter' ),
+					};
+					break;
+				case 'plan_selected':
+					taskData = {
+						title: translate( 'Free Plan' ),
```

Comment:
We may want to figure out something else here. Whether it is rename this as "select plan" or dynamically add the plan name?  Right now in testing with a site with an upgraded plan its a bit odd saying "Free Plan (Personal)" or "Free Plan (Business)":

![Screen Shot 2022-08-22 at 11 21 01 AM](https://user-images.githubusercontent.com/28742426/185957878-1f238126-58b1-42ef-8333-c1d9a09c47b0.png)

Reply:
I intentionally hardcoded 'Free Plan' here because I was focusing on other tasks and was going to handle it in another PR. I was planning on dynamically modifying the badge text (Personal or Business) and plan name here.